misk-aws/src/main/kotlin/misk/dynamodb/FakeDynamoDbModule.kt

```diff
@@ -0,0 +1,73 @@
+package misk.dynamodb
```

Comment:
We usually put fakes and testing-related code in a separate module so production code doesn't need to pull them in. See `misk-jobqueue-testing`, `misk-hibernate-testing`, etc

Reply:
THIS!!!!!!!!!!!!!!!!!!!!!